src/xo-app/host/tab-network.js

```diff
@@ -224,25 +314,16 @@ export default ({
     <Col>
       {!isEmpty(pifs)
         ? <span>
```

Comment:
You can remove the surrounding `<span>`.

Reply:
I'll create a new version with groupedActions and return directly the sortedTable as i did for logs tabs.